src/test/netbase_tests.cpp

```diff
@@ -321,4 +322,85 @@ BOOST_AUTO_TEST_CASE(netbase_parsenetwork)
     BOOST_CHECK_EQUAL(ParseNetwork(""), NET_UNROUTABLE);
 }
 
+BOOST_AUTO_TEST_CASE(netpermissions_test)
+{
+    std::string error;
+    NetWhitebindPermissions whitebindPermissions;
+    NetWhitelistPermissions whitelistPermissions;
+
+    // Detect invalid white bind
+    BOOST_CHECK(!NetWhitebindPermissions::TryParse("", whitebindPermissions, &error));
+    BOOST_CHECK(error.find("Cannot resolve -whitebind address") != -1);
```

Comment:
Suggest to use std::string::npos instead of -1 here and below (prevents "comparison between signed and unsigned" compiler warnings).